app/javascript/packs/mirador.js

```diff
@@ -15,11 +15,6 @@
 // const images = require.context('../images', true)
 // const imagePath = (name) => images(name, true)
 
-// Polyfills needed for IE and Edge
-import 'core-js/stable';
-import 'url-polyfill/url-polyfill';
-import 'unfetch/polyfill';
-
```

Comment:
do we no longer need these?

Reply:
IE is EOL and Edge never needed these.

Reply:
I wondered about Edge; OK to delete them in that case 👍🏻